src/main/resources/org/jenkinsci/plugins/mesos/MesosAdministrativeMonitor/message.jelly

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:i="jelly:fmt" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <div class="warning">
+        <p>Some Mesos slaves cannot be provisioned on the following labels:</p>
+        <ul>
+            <j:forEach items="${it.labels}" var="label">
+                <li><b>${label}</b></li>
```

Comment:
what about hyperlinks to label expressions? It allows to get usage stats for free